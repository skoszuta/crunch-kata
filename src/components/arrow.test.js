import React from 'react';
import { render } from 'enzyme';
import 'jest-styled-components';

import Arrow from './arrow';

describe('Arrow', () => {
  it('should render correctly', () => {
    const wrapper = render(
      <Arrow />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
