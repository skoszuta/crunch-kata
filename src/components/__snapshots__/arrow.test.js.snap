// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow should render correctly 1`] = `
.c0 {
  display: inline-block;
  width: 24px;
  height: 24px;
  background-image: url(test-file-stub);
}

<div
  class="c0"
/>
`;
